bin/annotate_gff.py

```diff
@@ -613,6 +629,163 @@ def get_ncrnas(ncrnas_file):
     return ncrnas
 
 
+def prepare_rna_gff_fields(cols):
+    rna_feature_name = "ncRNA"
+    if cols[1] in ["LSU_rRNA_bacteria", "SSU_rRNA_bacteria", "5S_rRNA"]:
+        rna_feature_name = "rRNA"
+    # ncRNA classes are described here: https://www.insdc.org/submitting-standards/ncrna-vocabulary/
+    ncrna_class = ""
+    rna_types = {
+        "antisense_RNA": [
```

Comment:
If you want to make this script a little easier to read you can add to a constants.py and import e.g. https://github.com/EBI-Metagenomics/genome_uploader/blob/main/genomeuploader/constants.py  then at the start from constants import "ANTISENSE_RNA". You can use that directly in the function then.